src/net/sourceforge/kolmafia/textui/command/ColdMedicineCabinetCommand.java

```diff
@@ -56,17 +81,211 @@ public static AdventureResult nextPill(Map<Character, Integer> counts) {
+    var consults = getConsultsUsed();
+    var turnsToNextConsult = getTurnsToNextConsult();
+
+    boolean guessing = true;
+    var output = new StringBuilder();
+
+    if (consults < 5) {
+      if (turnsToNextConsult > 0) {
+        output.append(turnsToNextConsult).append(" turns until next consult is ready.\n");
+      } else {
+        output.append("Consult is ready now");
+        guessing = KoLCharacter.inFightOrChoice();
+        if (guessing) {
+          output.append(" but can't visit right now so guessing your options...");
+        } else {
+          output.append("!");
+        }
+      }
+    }
+
+    return Map.entry(guessing, output);
+  }
+
+  public static SortedMap<String, AdventureResult> getCabinet() {
+    var guessing = shouldGuess();
+    return getCabinet(guessing.getKey());
+  }
+
+  public static SortedMap<String, AdventureResult> getCabinet(final boolean guessing) {
+    return guessing ? guessCabinet() : visitCabinet();
+  }
+
+  public static void status() {
+    var output = new StringBuilder();
+
+    var consults = getConsultsUsed();
+
+    output.append(consults).append("/5 consults used today.\n");
+
+    output.append("\n");
+
+    var g = shouldGuess();
+    output.append(g.getValue()).append("\n");
+    var guessing = g.getKey();
+    var cabinet = getCabinet(guessing);
+
+    if (cabinet == null) {
+      KoLmafia.updateDisplay(
+          KoLConstants.MafiaState.ERROR, "Cold Medicine Cabinet choice could not be parsed.");
+    } else {
+      output.append(formatItemList(cabinet, guessing));
+    }
 
     RequestLogger.printLine(output.toString());
   }
+
+  private static void collect(final int decision) {
+    if (getConsultsUsed() >= 5) {
+      KoLmafia.updateDisplay(
+          KoLConstants.MafiaState.ERROR, "You do not have any consults left for the day.");
+      return;
+    }
+
+    int turns = getTurnsToNextConsult();
+    if (turns > 0) {
+      KoLmafia.updateDisplay(
+          KoLConstants.MafiaState.ERROR, "You are not due a consult (" + turns + " turns to go).");
+      return;
+    }
+
+    RequestThread.postRequest(new CampgroundRequest("workshed"));
+    ChoiceManager.processChoiceAdventure(decision, "", true);
+  }
+
+  @Override
+  public void run(final String cmd, String parameter) {
+    var workshedItem = CampgroundRequest.getCurrentWorkshedItem();
+    if (workshedItem == null || workshedItem.getItemId() != ItemPool.COLD_MEDICINE_CABINET) {
+      KoLmafia.updateDisplay(
+          KoLConstants.MafiaState.ERROR, "You do not have a Cold Medicine Cabinet installed.");
+      return;
+    }
+
+    switch (parameter) {
+      case "" -> status();
+      case "equipment", "equip" -> collect(1);
+      case "food" -> collect(2);
+      case "booze", "wine" -> collect(3);
+      case "potion" -> collect(4);
+      case "pill" -> collect(5);
+      default -> {
+        KoLmafia.updateDisplay(KoLConstants.MafiaState.ERROR, "Paramter not recognised");
```

Comment:
```suggestion
        KoLmafia.updateDisplay(KoLConstants.MafiaState.ERROR, "Parameter not recognised");
```

Reply:
I spotted this and forgot to fix it - thank you